yarn.lock

```diff
@@ -9,11 +9,6 @@
   dependencies:
```

Comment:
I think this file should have significantly fewer changes than it currently does, especially around CDK packages.

Reply:
ill check , why this PR is adding  this many depedencies

Reply:
This was marked as resolved but it doesn't appear to be resolved.